tor_archivist/core/blossom.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+class ConfigurationError(Exception):
+    pass
+
+class BlossomAPI(object):
+    def __init__(
+            self,
+            email: str=None,
+            password: str=None,
+            api_key: str=None,
+            api_base_url: str="http://api.grafeas.localhost:8000",
+            login_url: str="http://grafeas.localhost:8000/login/"
+    ):
+        if not email:
+            raise ConfigurationError("Need an email address!")
```

Comment:
While I like the verbosity of a dedicated exception, I think this is precisely the use case intended for the built-in exception `ValueError()`

Reply:
I'll be honest, I did not review the blossom.py code too closely as it was provided by @itsthejoker so we'll have to get his input on this.

Reply:
@TheLonelyGhost you're totally right here, and it will have to be changed everywhere the file is used. Yes, let's go ahead and change this to ValueError.

Reply:
TBH, I didn't review a lot of @itsthejoker's code when he wrote this and it was pretty "experiment until it works for our purposes"-ish. I'm just reviewing what I see here, regardless of the original author. 😄